
Gcc-4.0 compile patch, imported from Fedora Core.

  - Rene Rebe <rene@exactcode.de>

--- emacs-21.3/lib-src/emacsclient.c~	2002-12-04 21:38:25.000000000 +0900
+++ emacs-21.3/lib-src/emacsclient.c	2005-03-18 20:21:50.542628565 +0900
@@ -38,7 +38,6 @@
 #endif /* not VMS */
 
 char *getenv (), *getwd ();
-char *getcwd ();
 
 /* This is defined with -D from the compilation command,
    which extracts it from ../lisp/version.el.  */
